### ISSUE TITLE
Test won't run or allow me to progress to the next step

### ISSUE THREAD

**Issue by chrissielavender** [FreeCodeCamp/FreeCodeCamp#11665]
Challenge [Target the same element with multiple jQuery Selectors](https://www.freecodecamp.com/challenges/target-the-same-element-with-multiple-jquery-selectors#?solution=%0Afccss%0A%20%20%24(document).ready(function()%20%7B%0A%24(%22button%22).addClass(animated)%3B%0A%20%20%20%20%24(%22.btn%22).addClass(shake)%3B%0A%20%20%20%20%24(%22%23target1%22).addClass(btn-primary)%3B%0A%20%20%20%20%24(%22%23target1%22).addClass(shake)%3B%0A%20%20%20%20%24(%22%23target1%22).addClass(animated)%3B%0A%20%20%0A%20%20%7D)%3B%0Afcces%0A%0A%3C!--%20Only%20change%20code%20above%20this%20line.%20--%3E%0A%0A%3Cdiv%20class%3D%22container-fluid%22%3E%0A%20%20%3Ch3%20class%3D%22text-primary%20text-center%22%3EjQuery%20Playground%3C%2Fh3%3E%0A%20%20%3Cdiv%20class%3D%22row%22%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23left-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22left-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target1%22%3E%23target1%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target2%22%3E%23target2%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target3%22%3E%23target3%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23right-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22right-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target4%22%3E%23target4%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target5%22%3E%23target5%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target6%22%3E%23target6%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%3C%2Fdiv%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  $(document).ready(function() {
$("button").addClass(animated);
    $(".btn").addClass(shake);
    $("#target1").addClass(btn-primary);
    $("#target1").addClass(shake);
    $("#target1").addClass(animated);
  
  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

```

**Comment by erictleung** [comment]
@chrissielavender Thank you for reporting this issue. Your code is incorrect and is preventing you from passing the challenge. You're missing quotes somewhere. Please first ask questions to the [Help Room](https://gitter.im/FreeCodeCamp/Help).

This is a standard message notifying you that this issue seems to be a request for help. Instead of asking for help here, please click the "help" button on the challenge on Free Code Camp, which will take you to the help chatroom for that specific challenge. You can also view our [full list of official chatrooms](http://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390).

If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding!

**Comment by chrissielavender** [comment]
Hi Eric,

When I first added the code the system updated with two check marks indicating my code was partially correct. I corrected the code add the additional lines, and then the section with the check marks were no longer visible. I refreshed and cleared my cache as instructed but the page still didn't load. I couldn't tell if the code was incorrect because the detail below the test buttons wouldn't load. 

Sincerely, 
Chrissie Lavender
